mkdocs.yml

```diff
@@ -198,4 +198,12 @@ nav:
         - Together AI: "api/llm/together.md"
         - Google Vertex AI: "api/llm/vertexai.md"
         - vLLM: "api/llm/vllm.md"
+    - Pipeline:
+      - "api/pipeline/index.md"
+      - Routing Batch Function: "api/pipeline/routing_batch_function.md"
+      - Typing: "api/pipeline/typing.md"
+      - Utils: "api/pipeline/utils.md"
     - CLI: "api/cli.md"
+  - Community:
+    - sections/community/index.md
+    - Issue dashboard: sections/community/popular_issues.md
```

Comment:
Not a big fan of this section! WDYT? @gabrielmbmb @plaguss

Reply:
I added it to align with Argilla and redirect some people to Slack/GitHub for participation and to feel a part of what we are doing. Might be less relevant for distilabel? If you guys have strong objections, let's remove it, otherwise, we can keep it.

Reply:
Popular issues can be a useful page to share on socials, even if it's not in the main menu.